src/index.html

```diff
@@ -1,13 +1,368 @@
-    <title>Miami</title>
+    <title>myBike</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="shortcut icon" href="./images/favicon.svg" type="image/x-icon">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#" class="logo">
+              <img
+              class="logo__image"
+              src="./images/MyBiKE.svg"
+              alt="MyBiKE">
+            </a>
+            <div class="header__menu">
+              <a
+              href="tel:+1 234 5555-55-55"
+              class="icon icon--phone header__menu-call">
+              </a>
+
+              <a
+              href="#menu"
+              class="icon icon--menu header__menu-opener">
+              </a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            Take the Streets
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/MyBiKE.svg"
+                alt="MyBiKE">
+            </a>
+
+            <a href="#"
+            class="icon icon__menu--crose">
+            </a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">about us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#compare" class="menu__link">Compare Bikes</a>
+            </li>
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+
+          <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__contacts menu__contacts--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+          href="tel:+1 234 5555-55-55"
+          class="menu__contacts menu__contacts--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+    <main class="main">
+      <section class="page__section about" id="about-us">
```

Comment:
Maybe main__section will be better )

```suggestion
    <main class="main">
      <section class="main__section about" id="about-us">
```

Reply:
renamed.